src/System.Collections/src/System/Collections/Generic/SortedList.cs

```diff
@@ -161,11 +161,11 @@ public SortedList(IDictionary<TKey, TValue> dictionary, IComparer<TKey> comparer
                 Debug.Assert(count == _keys.Length);
                 if (count > 1)
                 {
-                    Array.Sort<TKey, TValue>(keys, _values, comparer);
-                    comparer = Comparer; // obtain default if this is null.
+                    // Comparer is set to the default if the comparer parameter is null
+                    Array.Sort<TKey, TValue>(keys, _values, Comparer);
                     for (int i = 1; i != keys.Length; ++i)
                     {
-                        if (comparer.Compare(keys[i - 1], keys[i]) == 0)
+                        if (Comparer.Compare(keys[i - 1], keys[i]) == 0)
```

Comment:
Will the jitter optimise this, or will it do the n-1 field accesses this suggests?